vscode-mysql/v2/start.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 python3 ${VSCODE_USER}/coursera/submitButton.py
 /usr/local/bin/mysqlServer.sh mysqld &
-code-server --config /tmp/config/code-server/config.yaml 
+code-server --config /tmp/config/code-server/config.yaml --disable-update-check
```

Comment:
Can this setting be inside `config.yaml`?

Reply:
Maybe this ? 
https://stackoverflow.com/questions/42496935/disabling-visual-studio-code-update-notification/50657575

Reply:
Yup, it works in `config.yaml`. And "update.mode" is for VSCode updates, so it doesn't apply to code-server update notifications.